packages/contracts/scripts/test_userOp_testnet.ts

```diff
@@ -147,41 +149,98 @@ async function testConsecutiveUserOps(kernelAccount: SmartAccount, kernelClient:
 
     console.log("     Deployed successfully")
     console.log("\n[2/3] Sending UserOps...")
-    const hashes = []
+    const userOpsAndHashes = []
     const currentNonce = await kernelClient.account?.getNonce()
     if(!currentNonce){
         throw new Error("Failed to get nonce")
     }
     console.log("     Current nonce:", currentNonce)  
-    for(let i = 0; i < NUMBER_OF_USEROPS; i++){
-        const hash = await sendUserOp(kernelAccount, kernelClient, currentNonce + BigInt(i))
-        hashes.push(hash)
+    for(let i = 0; i < numberOfUserOps; i++){
+        const res = await sendUserOpNow(kernelAccount, kernelClient, currentNonce + BigInt(i))
+        userOpsAndHashes.push(res)
         
     }
-    console.log("\n[3/3] Awaiting receipts...")
+    // console.log("\n[3/3] Awaiting receipts...")
 
-    const statuses = await Promise.all(hashes.map(hash => pollForCompletion(hash)))
+    // const statuses = await Promise.all(userOpsAndHashes.map(userOpAndHash => pollForCompletion(userOpAndHash.hash)))
 
-    for(let i = 0; i < statuses.length; i++){
-        console.log(`UserOp ${i} status: ${statuses[i]}`)
-    }
-    console.log("Completed sending and awaiting receipts")
+    // for(let i = 0; i < statuses.length; i++){
```

Comment:
`pimlico_sendUserOperationNow` does not seem to allow for status polling, since it just goes immediately on-chain i guess